packages/nextjs/app/builders/0x4aa32F424d0E8fcF71B542B837ce804153773080/page.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import Image from "next/image";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const BuilderPage: NextPage = () => {
+  const size = { width: 32, height: 32 };
+  const iconStyle = "w-8 h-8 hover:scale-110 transition-transform duration-300 hover:drop-shadow-lg";
+
+  const anchorStyle = "text-2xl hover:scale-110 transition-transform duration-300 hover:drop-shadow-lg";
+
+  const icons = [
+    {
+      anchorUrl: "https://github.com/mufasa-0420",
+      anchorStyle,
+      iconUrl: "https://api.iconify.design/tabler:brand-github.svg",
+      iconAlt: "Github",
+      iconStyle,
+    },
+    {
+      anchorUrl: "https://t.me/bobbyliu0420",
+      anchorStyle,
+      iconUrl: "https://api.iconify.design/tabler:brand-telegram.svg",
+      iconAlt: "Telegram",
+      iconStyle,
+    },
+  ];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.